From ff0b066c33b9ecba3b42e2d2c43fd46b293a7213 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:23:57 -0400
Subject: [cpufreq] Kconfig: Fix spelling errors

Message-id: <1409235879-1283-144-git-send-email-prarit@redhat.com>
Patchwork-id: 89488
O-Subject: [RHEL7.1 PATCH BZ 1134639 143/185] cpufreq: Kconfig: Fix spelling errors
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 735dc2498bd97b22e465dd4054d1b7edb9f63ed2
Author: Stratos Karafotis <stratosk@semaphore.gr>
Date:   Tue Apr 22 22:40:10 2014 +0300

    cpufreq: Kconfig: Fix spelling errors

    Fix 4 spelling errors in help sections.

    Signed-off-by: Stratos Karafotis <stratosk@semaphore.gr>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Note, the last chunk is from upstream commit 8c88126.  I have
intentionally squashed that one line chunk in with this one.

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/Kconfig.x86 b/drivers/cpufreq/Kconfig.x86
index 6897ad8..89ae88f 100644
--- a/drivers/cpufreq/Kconfig.x86
+++ b/drivers/cpufreq/Kconfig.x86
@@ -10,7 +10,7 @@ config X86_INTEL_PSTATE
    The driver implements an internal governor and will become
           the scaling driver and governor for Sandy bridge processors.
 
-   When this driver is enabled it will become the perferred
+   When this driver is enabled it will become the preferred
           scaling driver for Sandy bridge processors.
 
    If in doubt, say N.
@@ -52,7 +52,7 @@ config X86_ACPI_CPUFREQ_CPB
  help
    The powernow-k8 driver used to provide a sysfs knob called "cpb"
    to disable the Core Performance Boosting feature of AMD CPUs. This
-   file has now been superseeded by the more generic "boost" entry.
+   file has now been superseded by the more generic "boost" entry.
 
    By enabling this option the acpi_cpufreq driver provides the old
    entry in addition to the new boost ones, for compatibility reasons.
@@ -129,7 +129,7 @@ config X86_AMD_FREQ_SENSITIVITY
  help
    This adds AMD-specific powersave bias function to the ondemand
    governor, which allows it to make more power-conscious frequency
-   change decisions based on feedback from hardware (availble on AMD
+   change decisions based on feedback from hardware (available on AMD
    Family 16h and above).
 
    Hardware feedback tells software how "sensitive" to frequency changes
-- 
1.7.1